packages/delegate/src/transforms/ExpandAbstractTypes.ts

```diff
@@ -75,11 +92,13 @@ function expandAbstractTypes(
   targetSchema: GraphQLSchema,
   mapping: Record<string, Array<string>>,
   reverseMapping: Record<string, Array<string>>,
```

Comment:
Maybe it would make sense to rename mapping to something more descriptive if there are multiple mappings?